components/Navbar.tsx

```diff
@@ -1,7 +1,7 @@
 import Link from "next/link";
 import ScrollToTop from "@/components/ScrollToTop";
 export default function Navbar() {
-  const isRegistered = true;
+  const isRegistered = false;
```

Comment:
can we get this value from the URL params until we finish up registration?

Reply:
Should a backend be written for it or not?